components/PostCard.tsx

```diff
@@ -1,13 +1,43 @@
+import {
+  isPostADraft,
+  isPostInTheFuture,
+} from "helpers/checkOfDraftOrFuturePost";
 import { PostDocumentWithoutContent } from "interfaces";
 import Image from "next/image";
 import Link from "next/link";
 import { useAppDispatch } from "redux/typesHooks";
 
 import { setTags } from "../redux/slices/selectedTags";
+
 interface Props {
   post: PostDocumentWithoutContent;
 }
 
+export const DraftPostMark = () => (
+  <div className="drt-post">
+    <div className="trn-drt trn trn-i">
+      <span className="trn-txt-drt trn-txt">draft</span>
+    </div>
+    <div className="sm-trns-drt">
+      <div className="sm-trns-i-drt-first trn sm-trns-i-drt sm-trns-i"></div>
+      <div className="sm-trns-i-drt-sec trn sm-trns-i-drt sm-trns-i"></div>
+    </div>
```

Comment:
WTF is drt?
Has this code been compiled? If so, it is not supposed to be read by a human

Otherwise, give it a human readable name

Reply:
I have written about this problem in our telegram chat. I don't understand, how I am supposed to change the name to the 
shorter in 2 or 3 times and save the readability of it.  I can delete all this classes, but then, we will have a problem of duplicates of styles

Reply:
@1alexvash

Reply:
@ntorbinskiy In telegram I answered to return previous classes, and in this comment I said to use "git revert" command for it, what's not clear on that one?